### PR TITLE
Feature tagging

### PR DESCRIPTION
**Pull request by The-Vector-007**

### CODE REVIEW

File: projects/shared-modules/mention/components/entity-profile/entity-profile.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <img *ngSwitchCase="'builds'" alt="{{ entity.name }}" src="{{ entity.images[0]?.i32 || '/assets/images/commudle-logo32.png' }}" title="{{ entity.name }}">
+    <img *ngSwitchCase="'channels'" alt="{{ entity.name }}" src="{{ entity.logo?.i32 || '/assets/images/commudle-logo32.png'  }}" title="{{ entity.name }}">
+    <img *ngSwitchCase="'communities'" alt="{{ entity.name }}" src="{{ entity.logo_image?.i32 || '/assets/images/commudle-logo32.png' }}" title="{{ entity.name }}">
+    <img *ngSwitchCase="'events'" alt="{{ entity.name }}" src="{{ entity.header_image?.i32 || '/assets/images/commudle-logo32.png' }}" title="{{ entity.name }}">
+    <img *ngSwitchCase="'labs'" alt="{{ entity.name }}" src="{{ entity.header_image?.i32 || '/assets/images/commudle-logo32.png' }}" title="{{ entity.name }}">
+  </div>
+  <div class="profile-info">
+
+    <p class="name {{ size }} d-flex">
+      <strong>{{ entity.name | slice:0:25 }}</strong>
```
Comment (Ashwin-op) [inline projects/shared-modules/mention/components/entity-profile/entity-profile.component.html]: Let's use truncate text pipe

File: projects/shared-modules/mention/components/entity-profile/entity-profile.component.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <img *ngIf="entity.is_expert" alt="Expert Tick" class="expert-icon" src="/assets/images/Expert-Tick.svg"
+           title="Expert Tick">
+    </p>
+
+    <p *ngIf="entity.type === 'users'" class="designation {{ size }}">
+      {{ entity.username | slice:0:25 }}
```
Comment (Ashwin-op) [inline projects/shared-modules/mention/components/entity-profile/entity-profile.component.html]: Let's use truncate text pipe

File: projects/shared-modules/mention/components/entity-profile/entity-profile.component.scss [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+  .online-status {
+    position: absolute;
+    top: 6px;
+    left: 3px;
+    width: 10px;
+    height: 10px;
+    border-radius: 50%;
+
+    &.online {
```
Comment (Ashwin-op) [inline projects/shared-modules/mention/components/entity-profile/entity-profile.component.scss]: Are these styles being used in the corresponding HTML file? If not, let's remove it

File: projects/shared-modules/mention/components/suggestion-box/suggestion-box.component.scss [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+.mention-dropdown {
+  bottom: 100%;
+  top: auto;
+  margin-bottom: 2px;
+  background: white;
+}
+
+.selected{
+  background-color: cornflowerblue;
```
Comment (Ashwin-op) [inline projects/shared-modules/mention/components/suggestion-box/suggestion-box.component.scss]: Let's use variables for colors, if the color you need doesn't have a variable, then create one

File: projects/shared-modules/mention/directives/mention-directive/mention.directive.spec.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import { MentionDirective } from './mention.directive';
+
+describe('MentionDirective', () => {
+  it('should create an instance', () => {
```
Comment (Ashwin-op) [inline projects/shared-modules/mention/directives/mention-directive/mention.directive.spec.ts]: We don't need this file, let's delete it

File: projects/shared-modules/mention/service/mention.service.spec.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { MentionService } from './mention.service';
+
```
Comment (Ashwin-op) [inline projects/shared-modules/mention/service/mention.service.spec.ts]: We don't need this file, let's delete it

File: projects/shared-modules/mention/mention.module.ts [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    //Nebular
+    NbListModule
+  ],
+  
+  exports: [
+    MentionDirective
+  ],
+
+  entryComponents: [
+    SuggestionBoxComponent,
```
Comment (Ashwin-op) [inline projects/shared-modules/mention/mention.module.ts]: Why have we added these components to this property?

**Review by Ashwin-op — CHANGES_REQUESTED** [review]

File: projects/shared-modules/mention/components/entity-profile/entity-profile.component.html (hunk 1 above)
Comment (The-Vector-007) [reply projects/shared-modules/mention/components/entity-profile/entity-profile.component.html]: done!

**Pull request merged**